.spelling

```diff
@@ -596,6 +602,8 @@ subnetworks
 subresource
 subresources
 substring
+suffix
```

Comment:
surprised this is not in the dictionary - are you certain?

Reply:
I don't see it https://github.com/istio/istio.io/blob/master/.spelling#L598

Reply:
Istio uses a default dictionary. The `.spelling` file is an extension to the default dictionary. I am just surprised to see `suffix` was needed to extend the default dictionary. I'd suggest reworking this document to remove all references in `.spelling` by using backticks. If something is required, you can keep it in `.spelling`.